docs/reference/modules.rst

```diff
@@ -13,6 +13,7 @@ Modules
    interpolate
    logging
    optimize
+   rc
```

Comment:
Cryptic module name?

Reply:
Is this even something we want to be documented here as core user-facing API? Is there a better place?

Reply:
> Cryptic module name?

Kinda. I wanted to avoid an underscore as in `runtime_config`. Any better ideas?

> Is this even something we want to be documented here as core user-facing API? Is there a better place?

Agree that it should not be too visible / clutter the module list. But it needs to be somewhere. What about turning the current 'Runtime Configuration' notebook into a subsection next to the module docs? I.e. 
```
top level > Runtime Configuration > {guide (notebook), module}
```
This would be a bit odd because it separates the module docs. That makes it a bit harder to see the actual structure.

Given that the Runtime Configuration reference is at the top level, it might be ok to also but the `rc` module where it is right now.

Reply:
> Kinda. I wanted to avoid an underscore as in runtime_config. Any better ideas?

`config`? Do we have anything user-facing that is not runtime?

> Agree that it should not be too visible / clutter the module list. But it needs to be somewhere. What about turning the current 'Runtime Configuration' notebook into a subsection next to the module docs? I.e.

Is there a way to have the module docs at the end of the current notebook? Similar to how h5py does it: https://docs.h5py.org/en/stable/high/dataset.html (probably they are not using notebooks though?).

Reply:
I doubt it. But I will try.

Reply:
Called it 'configuration' and split the module docs page into subsections to take focus away from the rc and logging docs.